src/bootloader.rs

```diff
@@ -223,6 +228,6 @@ impl Bootloader {
 fn test_all_properties() {
     // let (vid, pid) = (0x1fc9, 0x0021);
     let (vid, pid) = (0x1209, 0xb000);
-    let bootloader = Bootloader::try_new(vid, pid).unwrap();
+    let bootloader = Bootloader::try_new(Some(vid), Some(pid)).unwrap();
```

Comment:
Should make this a test that runs behind a feature flag "test-with-attached-device" or similar, so we can run the tests in CI.